internal/endpoints/payloads.go

```diff
@@ -158,6 +159,7 @@ func RequestIdPayloads(w http.ResponseWriter, r *http.Request) {
 
 	reqID := chi.URLParam(r, "request_id")
 	sortBy := r.URL.Query().Get("sort_by")
+	verbosity := r.URL.Query().Get("verbosity")
```

Comment:
I think if we do a `var verbosity string = "0"`, we can just overwrite it if we get a query for it and we can remove the "0" case above. So this line would be the same but would be just `=` instead of `:=`

Reply:
gotcha!!  I'll go ahead and change that up!  :)